tests/core/parameters_fancy_indexing/coefficient_de_minoration.yaml

```diff
@@ -0,0 +1,135 @@
+description: Coefficient de minoration ARRCO
```

Comment:
This parameter isn't used, is it?
Was it used as an example to explicit some use cas of the `asof` parameter as it doesn't contain `before...` and `after...` keys? Add it in the test or remove the file if it is no longer used?

Reply:
Yes could be removed